contrib/completion/zsh/_docker

```diff
@@ -788,7 +788,7 @@ __docker_node_commands() {
         "ls:List nodes in the swarm"
         "promote:Promote a node as manager in the swarm"
         "rm:Remove one or more nodes from the swarm"
-        "ps:List tasks running on a node"
+        "ps:List tasks running on a node, defaults to self"
```

Comment:
Thanks for your advice. @yanheven
@stevvooe @thaJeztah WDYT?
And both are OK for me.